src/System.cpp

```diff
@@ -83,6 +96,18 @@ namespace System
         }
     }
 
+    std::string GetWorkspaceSymbol(int index) {
+        if (index < 0 || index > 9) {
+            return "";
```

Comment:
This should either ASSERT, or LOG an error, since it is never expected to be in that range and clearly an error from the programmer.